Master-Game/World/Map.cs

```diff
@@ -39,5 +46,78 @@ public void ResetTiles()
                 }
             }
         }
+
+        public MapData LoadJson()
+        {
+            string fileName = "Jay01.json";
+            string filePath = Path.GetDirectoryName(Path.GetDirectoryName(System.IO.Directory.GetCurrentDirectory())) + "\\Assets\\Maps\\" + fileName;
+            using (StreamReader reader = new StreamReader(filePath))
+            {
+                string jsonStringData = reader.ReadToEnd();
+
+                if (!jsonStringData.Equals(""))
+                {
+                    return JsonConvert.DeserializeObject<MapData>(jsonStringData);
+                }
+                else
+                {
+                    return null; //String data value cannot be empty.
+                }
+                
+            }
+        }
+
+        public class MapData
+        {
+            public string name { get; set; }
+            public string version { get; set; }
+            public int rows { get; set; }
+            public int cols { get; set; }
+            public Tiles [] tiles { get; set; }
+            public Entities[] entities { get; set; }
+        }
+
+        public class Tiles
+        {
+            public int type;
+
```

Comment:
no need for new lines between each parameter

Reply:
Done.